### DISCUSSION TITLE
Is registration on UMS forum closed?

### DISCUSSION THREAD

**Discussion by kreon28** [UniversalMediaServer/UniversalMediaServer#2414] [Q&A] [answered]
Hi,

Sorry for the noise here, but I have an user question which is perfect for forum. Unfortunately, it looks like registration on forum is blocked.

When submitting user data, browser says : "This site key is not enabled for the invisible captcha." and registration is on hold.
Could you fix that?

**Comment by SubJunk** [comment]
@kreon28 thanks for reporting this, I will look into it now

**Comment by SubJunk** [comment, marked as answer]
I confirmed the bug and have hopefully fixed it now. Thanks again!

**Reply by kreon28** [reply]
Yes. It is working now! Thanks for fixing this at short notice.
